### PR TITLE
DPNG-12495 - revise model api for new scoring engine

### PR DESCRIPTION
**Pull request by jitendra42**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,25 +108,46 @@ The trained model can also be exported to a .mar file, to be used with the scori
...
-```
+* revise model via .mar file
+
+	```
+	curl -i -X POST  -F file=@revised_model.mar http://localhost:9100/v2/reviseMarFile
+	```
+
+* revise model via byte stream of model file 
+
+	```
```
Comment (blbarker) [inline README.md]: Should mention this example is Python

File: README.md [hunk 2]
```diff
@@ -108,25 +108,46 @@ The trained model can also be exported to a .mar file, to be used with the scori
...
+	modelBytes = open('./revised_model.mar', 'rb').read()
+	requests.post(url='http://localhost:9100/v2/reviseMarBytes', data=modelBytes, headers={'Content-Type': 'application/octet-stream'})
+	```
+
+Forcefully revising incompatible model:
```
Comment (blbarker) [inline README.md]: Explain what incompatible means --i.e. results in a different scoring API

File: model-scoring-core/src/main/scala/org/trustedanalytics/scoring/ScoringService.scala [hunk 3]
```diff
@@ -1,97 +1,103 @@
...
   lazy val description = {
     new ServiceDescription(name = "Trusted Analytics",
       identifier = "ia",
       versions = List("v1", "v2"))
   }
 
-  val jsonFormat = new ScoringServiceJsonProtocol()
-  import jsonFormat._
+  /**
+    * Case class to hold model, model tar file path and json format protocol object
```
Comment (blbarker) [inline model-scoring-core/src/main/scala/org/trustedanalytics/scoring/ScoringService.scala]: Looks like it doesn't hold the tar file path.

Comment (jitendra42) [inline model-scoring-core/src/main/scala/org/trustedanalytics/scoring/ScoringService.scala]: Yes, that's correct. The latest scoring engine do not take path from HDFS, hence we no longer hold the tar file path.

**Pull request merged**